pkg/cli/cliflags/flags.go

```diff
@@ -827,8 +827,8 @@ Also, if you use equal signs in the file path to a store, you must use the
 	StorageEngine = FlagInfo{
 		Name: "storage-engine",
 		Description: `
-Storage engine to use for all stores on this cockroach node. Options are pebble
-or rocksdb. If unspecified, pebble is used.
+Storage engine to use for all stores on this cockroach node. The only option is pebble. Deprecated;
```

Comment:
Would it make sense to remove the flag entirely?

Reply:
I've kept it around for now just so anyone forcibly using `--storage-engine=pebble` doesn't see it fail rightaway. But it's probably a good idea to remove it entirely.

Reply:
probably makes sense to keep it for one more version, so that people migrating don't have to change flags in between right?

Reply:
Agreed on keeping the flag for the 21.1 release. We may even want to support `--storage-engine=rocksdb` with a loud warning that you're actually using Pebble instead.